src/ai/backend/manager/models/vfolder.py

```diff
@@ -888,22 +888,23 @@ async def prepare_vfolder_mounts(
     requested_mounts: list[str] = [
         name for name in requested_mount_references if isinstance(name, str)
     ]
-    requested_mount_map: dict[str, str] = {
+    requested_mount_name_map: dict[str, str] = {
         name: path for name, path in requested_mount_reference_map.items() if isinstance(name, str)
```

Comment:
The only difference between `requested_mount_references` and `requested_mount_reference_map` is that one is a list and the other is a dict, right?

Reply:
The value of `requested_mount_reference_map` dictionary indicates the destination directory of vfolder mounts